### PR TITLE
Adding unttests for example notebooks

### PR DESCRIPTION
**Pull request by bschifferer**
Adding unittests with testbook for notebooks (currently 01, 02 and 06). Other notebooks will follow with next PR.

https://github.com/NVIDIA-Merlin/Merlin/issues/205

### CODE REVIEW

**Comment by bschifferer** [comment]
rerun tests

File: merlin/datasets/entertainment/movielens/1m-raw/ratings/schema.pbtxt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+feature {
+  name: "rating"
+  type: INT
+  int_domain {
+    name: "rating"
+    max: 6
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
```
Comment (marcromeyn) [inline merlin/datasets/entertainment/movielens/1m-raw/ratings/schema.pbtxt:46]: Should we tag this as target?

**Review by marcromeyn — APPROVED** [review]

File: merlin/datasets/entertainment/movielens/1m-raw/ratings/schema.pbtxt (hunk 1 above)
Comment (bschifferer) [reply merlin/datasets/entertainment/movielens/1m-raw/ratings/schema.pbtxt:46]: This schema represents the raw data for unittests. The raw data does not provide any tags. I wanted to use as few tags as possible. Actually, we should remove the tags from all columns, but then the data generator did not work.

**Pull request merged**